errbot/backends/slack.py

```diff
@@ -394,17 +417,21 @@ def _message_event_handler(self, event):
 
     def userid_to_username(self, id_):
         """Convert a Slack user ID to their user name"""
-        user = [user for user in self.sc.server.users if user.id == id_]
+        users = self.api_call('users.list')
```

Comment:
what is the difference between self.sc.server.users and self.api_call('users.list') ?